backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Program.cs

```diff
@@ -72,6 +73,7 @@
     app.UseHttpsRedirection();
 }
 
+app.UseDefaultSecurityHeaders();
```

Comment:
We have to introduce a blank line after this line  to resolve the error CSC : error SA1516: Elements should be separated by blank line